config/chime_science_run_gpu.yaml

```diff
@@ -32,7 +32,7 @@ log_level: info
 num_elements: 2048
 num_local_freq: 1
 num_data_sets: 1
-samples_per_data_set: 49152
+samples_per_data_set: 512
```

Comment:
@chitrangpatel can you revert this?  I'm not sure how it got into this PR, but it's clearly wrong.

Reply:
I think Davor had set it to 49152 while he was converting and we changed it to 512 since thats what we are currently using. Should I revert or leave it as is?

Reply:
Revert to develop, it should be `49152`

Reply:
Sorry revert to the parent branch, develop will have other changes as well.